src/app/config/confparse.c

```diff
@@ -534,6 +534,22 @@ config_var_is_settable(const config_var_t *var)
   return struct_var_is_settable(&var->member);
 }
 
+/**
+ * Return true iff the controller is allowed to fetch this variable.
```

Comment:
It might be clearer if we say:
```
iff the controller is allowed to fetch the value of <b>var</b>
```
Because
* is_listable() is about fetching names
* most other docs in this module use `<b>var</b>` not "this variable"

Reply:
9f833dce93f076 makes these changes.

Reply:
.